CHANGES.txt

```diff
@@ -50,7 +52,6 @@ version 2.0.0.
 
 - ops.cascaded_union
 - geometry .empty()
-- geometry .to_wkb() and .to_wkt()
```

Comment:
Those were actually removed and not deprecated, so updated the note (they were deprecated long time ago, see https://github.com/Toblerity/Shapely/pull/974)